Emby.Server.Implementations/Updates/InstallationManager.cs

```diff
@@ -190,6 +189,22 @@ public async Task<IReadOnlyList<PackageInfo>> GetAvailablePackages(CancellationT
                             continue;
                         }
 
+                        for (var i = package.versions.Count - 1; i >= 0; i--)
```

Comment:
Is it necessary to loop through them all? If they are pre-sorted, then it's a reasonable assumption that a targetAbi that is lower than or equal to the app version means we can break out.

Reply:
I suppose custom repos won't necessarily adhere to those "rules".

Reply:
Correct, custom repos could potentially have the versions in any order